### ISSUE TITLE
FloaterProps missing disableFlip prop in type definition

### ISSUE THREAD

**Issue by deduskaa** [gilbarbara/react-joyride#586]
I'm getting this error while trying to use disableFlip in step configs floaterProps. Would it be possible to add disableFlip prop to the type definitions ?

```
Type '{ disableAnimation: true; disableFlip: true; }' is not assignable to type 'FloaterProps'.
  Object literal may only specify known properties, and 'disableFlip' does not exist in type 'FloaterProps'.ts(2322)
index.d.ts(173, 3): The expected type comes from property 'floaterProps' which is declared here on type 'Step'
```

**Comment by MovingGifts** [comment]
Same issue... any solution to this? When I set the offset to a certain negative number, it flips the placement from bottom to top, but it should remain as bottom. Any idea how to get `disableFlip` or the intended behavior with the library as is?

**Comment by gilbarbara** [comment]
@MovingGifts Are you having trouble with Typescript or the negative number?
If it is the latter, please check popper.js docs for v1

**Comment by MovingGifts** [comment]
@gilbarbara The issue is with the negative number `offset: -20`. When I use:

```
{
  target: '.my-tip-anchor',
  content: 'Tap here for more',
  disableOverlay: false,
  disableCloseOnEsc: true,
  spotlightClicks: true,
  disableBeacon: true,
  hideFooter: false,
  // offset: -10, // offset needs to be higher negative value, but that flips the direction when we go over -10...
  offset: -20, // this is a good offset value, but that flips the direction of the tooltip from bottom to top...
  placement: 'bottom',
  styles: {
    tooltip: {
      width: 300,
      padding: 10
    },
    tooltipContent: {
      padding: 0,
      fontSize: 16,
    },
    buttonClose: {
      display: 'none',
    },
  }
}
```

When offset is `-10` placement bottom, the tooltip is still too far from the target. When offset is `-20` placement bottom, to make it closer up to the target, the tooltip is rendered as if the placement was set to top, not sure why or how to prevent that as it should always respect the placement: 'bottom' for this specific tooltip and not try to flip it, just get closer up to the target vertically.

I looked at the popper.js v1 api here [https://popper.js.org/docs/v1/#modifiers..flip](url), but still not sure how to apply that to the above json to get it to not flip with `offset: -20`. Any idea what we need to change to fix it?

**Comment by gilbarbara** [comment]
Hey @MovingGifts 

So, it's not the same issue at all, @deduskaa is talking about Typescript declaration, not behavior.

Did you check the [docs](https://docs.react-joyride.com/)?
You probably need to set the `floaterProps` prop.
```
{
  floaterProps: { disableFlip: true }
}
```

**Comment by MovingGifts** [comment]
@gilbarbara Thank you so much, that did the trick!

**Comment by gilbarbara** [comment]
Fixed in cfe6a6ee800dc73f2381dec3f7ce801beb98c0f1
